example/android/app/src/main/java/com/adyenexample/MainActivity.java

```diff
@@ -24,7 +28,7 @@ protected String getMainComponentName() {
 
   @Override
   protected void onCreate(Bundle savedInstanceState) {
-    super.onCreate(savedInstanceState);
+    super.onCreate(null);
```

Comment:
I am wondering why is this necessary?

Reply:
You can check here: https://reactnavigation.org/docs/7.x/getting-started
and more detailed here: https://github.com/software-mansion/react-native-screens/issues/17